docs/framework/wcf/feature-details/security-protocols-version-1-0.md

```diff
@@ -62,9 +62,9 @@ The Web Services Security Protocols provide Web services security mechanisms tha
   
 |Prefix|Namespace|  
 |------------|---------------|  
-|s|<https://www.w3.org/2003/05/soap-envelope/>|
+|s|<http://www.w3.org/2003/05/soap-envelope/>|
```

Comment:
ℹ️ These are reverted to http per request re: schemas.

Reply:
Thank you!